cmd/main.go

```diff
@@ -55,6 +57,12 @@ func init() {
 		log.Fatalf("session cache init failed, reason: %v", err)
 	}
 	session.SessionCache = sessionCache
+
+	backend, err := storage.NewBackend(conf.Archive)
+	if err != nil {
+		log.Fatal(err)
+	}
```

Comment:
```suggestion
	if err != nil {
		log.Fatalf("Error initiating storage backend, reason: %v", err)
	}
```

Reply:
When we throw a `fatal` it is nice to have some extra information since the given errors might not be so helpful.